src/features/ar.ts

```diff
@@ -56,9 +60,12 @@ export const openARViewer = (() => {
 
     title = encodeURIComponent(title);
     modelUrl.protocol = 'intent://';
+    // It's possible for a model URL to have meaningful query parameters
+    // already. Sure hope they aren't called 'link' or 'title' though 😅
+    modelUrl.search +=
+        (modelUrl.search ? '&' : '') + `link=${link}&title=${title}`;
```

Comment:
Can we log a console warning if it does find a link or title query param?

Reply:
Good idea!

Reply:
Done.